model/MESSAGE_AND_TOKEN_FORMAT.md

```diff
@@ -65,82 +56,41 @@ The most up-to-date place to look for code is here:
-
-{bot_name}:
-```
-
-Example:
-
-```
-You are a helpful assistant called Joi trained by OpenAssistant on large corpus of data, you will now help user to answer the question as concise as possible
-User: hello!
-
-Joi:
-```
-
 ## Message Format v2
 
-**Note:** There is a variation where the `<prefix>` and `</prefix>` tags are
-omitted, leaving only the `pre_text`. This will be specified in the
-`config.add_prefix_tokens`.
-
-**Note:** That `<prefix>` refers to a specific token that the model knows about.
-If the user literally typed `<prefix>` then that would be tokenized to
-completely different non-special tokens, reducing possible attacks.
+This is used by most Open Assistant models.
 
 Format:
 
-```xml
-<prefix>{pre_text}</prefix><human>{prompt}<bot>
 ```
-
-Example:
-
-```xml
-<prefix>You are a helpful assistant called Joi trained by OpenAssistant on large corpus of data, you will now help user to answer the question as concise as possible</prefix><human>hello!<bot>
+<|prompter|>{prompt}<|endoftext|><|assistant|>
 ```
 
-Model will then reply, padding the ending with **zero or more** `<|endoftext|>`.
-This is just to make entries in a batch the same size.
-
-## Message Format v2.5 old
-
-**Note:** There is a variation where the `<|prefix_begin|>` and `<|prefix_end|>`
-tags are omitted, leaving only the `pre_text`. This will be specified in the
-`config.add_prefix_tokens`.
-
-**Note:** That `<|prefix_begin|>` etc are special tokens, and refers to a
-specific special token that the model knows about. If the user literally typed
-`<|prefix_begin|>` then that would be tokenized to completely different
-non-special tokens, reducing possible attacks.
-
-Format:
-
-```
-<|prefix_begin|>{pre_text}<|prefix_end|><|prompter|>{prompt}<|endoftext|><|assistant|>
-```
+There is no specific prefix tag. A prefix could be placed before the first
+prompter message or in the first prompter message.
 
 Example:
 
 ```
-<|prefix_begin|>You are a large language model that wants to be helpful<|prefix_end|>
+You are a large language model that wants to be helpful<|prompter|>Hello!<|endoftext|><|assistant|>
 ```
 
 Model will then reply, padding the ending with **zero or more** `<|endoftext|>`.
 This is just to make entries in a batch the same size.
 
-## Message Format v2.5-new (Not currently committed at time of writing)
+## Message Format v2-new
+
+Experiments are ongoing with new Open Assistant models using this format.
 
 **Note:** The `<|system|>{prefix}<|endoftext|>` is omitted entirely if `prefix`
-is empty. **Note:** I've added newlines and comment just for readability here.
-They aren't in the format.
+is empty.
+
+**Note:** I've added newlines and comment just for readability here. They aren't
+in the format.
+
+**Note**: That `<|system|>` refers to a specific token that the model knows
+about. If the user literally typed `<|system|>` then that would be tokenized to
+completely different non-special tokens, reducing possible attacks.
```

Comment:
hm. Escaping of tokens in actual messages would be a good feature, currently I think this is not the case and  if `<|system|>` would appear in the text it would be translated to the special-system token by the tokenizer.

Reply:
Ok, this was in the original doc, guess it was optimistic :D Will remove this